ansible/roles/gluster/tasks/install_gluster.yml

```diff
@@ -0,0 +1,59 @@
+---
+
+- name: install glusterfs repository
+  yum: name=centos-release-gluster state=present
+
+- name: install glusterfs
+  yum: name=glusterfs-server state=present
+
+- name: configure gluster brick device
+  filesystem:
+    dev: "{{ gluster_device }}"
+    fstype: xfs
+    opts: "-i size=512"
```

Comment:
either way we can always make this variable later. Honestly, how to deploy it for production will be @vvb and @mapuri's responsibility, not as much ours.